experimental/Schemes/ProjectiveSchemes.jl

```diff
@@ -1014,3 +1014,30 @@ of ``X`` and ``Y``, respectively.
   ff = CoveredSchemeMorphism(X, Y, phi)
   return ff
 end
+
+# Basic functionality required for Warham
+@attr Integer function dim(P::AbsProjectiveScheme{<:Field})
```

Comment:
There is no benefit for type stability in providing an abstract type. 
It is not always an int?

Reply:
Who knows... It can change in the future. Maybe one day people even return `ZZRingElem`.

Reply:
This serves multiple purposes:
- it documents the expected return type ;-)
- if some method returns something else, e.g. a Singular integer, we get an error, which is *good*
- and lastly, type stability.

So yeah, `Int` sounds good to me. If someone truly needs to work with schemes in dimension > 2^63, I think we have other problems to solve first to make that possible...